source/server/hot_restart_impl.cc

```diff
@@ -53,7 +53,11 @@ SharedMemory& SharedMemory::initialize(uint64_t stats_set_size, Options& options
 
     // If we are meant to be first, attempt to unlink a previous shared memory instance. If this
     // is a clean restart this should then allow the shm_open() call below to succeed.
-    os_sys_calls.shmUnlink(shmem_name.c_str());
+    const Api::SysCallIntResult result = os_sys_calls.shmUnlink(shmem_name.c_str());
```

Comment:
@knightXun It is intended to ignore the result and not panic, please read the comment above. Tests are failing.

Reply:
thanks for your remind.

Reply:
question: may ignoring this result bring some problem?